### PR TITLE
Explore proofread: Airnode rewrite (partial)

### PR DESCRIPTION
**Pull request by martinkolenic**

### CODE REVIEW

**Comment by Siegrift** [comment]
I think you wanted to wait for @wkande or @dcroote approval 😄 

Btw. @wkande make sure to set up branch protection for the main branch.

**Comment by dcroote** [comment]
😅 

@wkande I also added branch protection (require a pull request before merging) for `live-site`, but let me know if you disagree

File: docs/explore/airnode/implementation.md [hunk 1]
```diff
@@ -14,57 +14,60 @@ tags:
...
+an additional $100, which means that using the operation multiple times would
+have a different effect than using it once.
+
+Stateless oracle nodes cannot "remember" if they already made an API call
+associated with a given request, and, under certain conditions, they may repeat
+it. At present, this is not an issue since oracles are only used for performing
+idempotent operations. In the future, however, Airnode intends to support
+non-idempotent operations as well. We are currently researching alternative
+methods of achieving this while protecting the resiliency provided
+bystatelessness.
```
Comment (Siegrift) [inline docs/explore/airnode/implementation.md:70]: The `bystatelessness` word feels too complex.

**Review by Siegrift — COMMENTED** [review]
👍 LGTM. 

I find your updates more readable and concise, left just one comment.

File: docs/explore/airnode/implementation.md (hunk 1 above)
Comment (wkande) [reply docs/explore/airnode/implementation.md:70]: @martinkolenic : Do you want to do something with `bystatelessness`?

Comment (dcroote) [reply docs/explore/airnode/implementation.md:70]: This should be two words `by statelessness`

Comment (wkande) [reply docs/explore/airnode/implementation.md:70]: @dcroote  - @martinkolenic : I will make the change suggested by Derek.

Comment (martinkolenic) [reply docs/explore/airnode/implementation.md:70]: Hi, @dcroote and @wkande sorry for the late reply, I was, indeed, off due to illness. The extra long word was a mistake on my part, no idea why it got past the spellchecker. Thanks for sorting that out. I'll review the rest of the Explore section and then I'll reach out to you, Warren, to discuss any other documents I could work on while I'm still on the project.

**Pull request merged**